wiki/en/Running-a-Server.md

```diff
@@ -201,15 +201,15 @@ Jamulus can be run in Server mode from the desktop. This gives you a graphical u
 
 **None**: By default, you will not be connected to a Directory and will be running in unregistered mode. [Read these instructions](#running-an-unregistered-server) to have other people connect to your Server in this mode.
 
-**Genre**: To allow other people to see your Server on one of the built-in Public Directories, select your desired genre Directory. You should see a confirmation message saying whether your Server has registered successfully. If not, and you leave your Server running, it will keep trying to register until a free slot becomes available.  
+**Genre**: To allow other people to see your Server on one of the Directories built into the Client, select your desired genre Directory. You should see a confirmation message saying whether your Server has registered successfully. If not, and you leave your Server running, it will keep trying to register until a free slot becomes available.
```

Comment:
Should we use 'built into Jamulus' instead of 'built into the client'?

Reply:
> Should we use 'built into Jamulus' instead of 'built into the client'?

That wouldn't make any sense.  We're talking about Jamulus in this document - running as a Server.  Here, we're explicitly wanting to talk about Jamulus running as a Client.  How would removing that distinction help?

Reply:
Re reading the sentence, it's clear now and it's like that. Sorry for the noise. :-)